.travis.yml

```diff
@@ -0,0 +1,31 @@
+dist: trusty
+
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - wget
+      - pkg-config
+
+before_install:
+  - wget https://github.com/bazelbuild/bazelisk/releases/download/v1.6.0/bazelisk-linux-amd64
+  - echo "616f65bcdcfd134a19d5d86c591c35098d7732be26145bf06f02ec9e3e52700c bazelisk-linux-amd64" | sha256sum --check
+  - sudo mv bazelisk-linux-amd64 /usr/bin/bazelisk
```

Comment:
copy to local path relative to working directory, then modify path. e.g. like https://docs.travis-ci.com/user/installing-dependencies/#installing-projects-from-source

Reply:
I hope you don't mind my current solution :)